lib/govuk_publishing_components/app_helpers/taxon_breadcrumbs.rb

```diff
@@ -15,7 +15,7 @@ def breadcrumbs
           {
             title: parent.title,
             url: parent.base_path,
-            is_page_parent: index.zero?
+            is_page_parent: index.zero?,
```

Comment:
Probably not necessary here but I doubt it'll break anything (unless it breaks any linter tests)

Reply:
Yes, it's a linting thing